js/sales.js

```diff
@@ -123,6 +123,65 @@
         return dataCell;
     }
 
+    function getNewStore(event){
+        event.preventDefault();
+        const form = event.target;
+        const storeName = form.querySelector('#store-name').value;
+        const minHourlyCustomers = parseInt(form.querySelector('#min-hourly-customer').value);
```

Comment:
If interested, it looks like if you give a `name` attribute to your form `input` elements, you can reference them as properties on `event.target`. E.g., `name="store-name"` can be referenced as `event.target.name`.

Not sure if it's any faster than using `querySelector`, but it might be a little more concise.